servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1419,88 +1421,91 @@ public final Completable liftAsync(CompletableOperator operator) {
     }
 
     /**
-     * Creates a new {@link Completable} that will use the passed {@link Executor} to invoke all {@link Subscriber}
-     * methods.
-     * This method does <strong>not</strong> override preceding {@link Executor}s, if any, specified for {@code this}
-     * {@link Completable}. Only subsequent operations, if any, added in this execution chain will use this
-     * {@link Executor}.
+     * Creates a new {@link Completable} that will use the passed {@link io.servicetalk.concurrent.Executor} to invoke
+     * all {@link Subscriber} methods.
+     * This method does <strong>not</strong> override preceding {@link io.servicetalk.concurrent.Executor}s, if any,
+     * specified for {@code this} {@link Completable}. Only subsequent operations, if any, added in this execution chain
+     * will use this {@link io.servicetalk.concurrent.Executor}.
      * <p>
-     * Note: unlike {@link #publishOn(Executor, BooleanSupplier)}, current operator always enforces offloading to the
-     * passed {@link Executor}.
+     * Note: unlike {@link #publishOn(io.servicetalk.concurrent.Executor, BooleanSupplier)}, current operator always
+     * enforces offloading to the passed {@link io.servicetalk.concurrent.Executor}.
      *
-     * @param executor {@link Executor} to use.
-     * @return A new {@link Completable} that will use the passed {@link Executor} to invoke all {@link Subscriber}
-     * methods.
-     * @see #publishOn(Executor, BooleanSupplier)
+     * @param executor {@link io.servicetalk.concurrent.Executor} to use.
+     * @return A new {@link Completable} that will use the passed {@link io.servicetalk.concurrent.Executor} to invoke
+     * all {@link Subscriber} methods.
+     * @see #publishOn(io.servicetalk.concurrent.Executor, BooleanSupplier)
      */
-    public final Completable publishOn(Executor executor) {
+    public final Completable publishOn(io.servicetalk.concurrent.Executor executor) {
```

Comment:
Decided that we don't need to go through the deprecated approach because these methods are backward compatible for the compilation. We do not need ABI compatibility for 0.42.